src/test/java/com/wikia/webdriver/common/core/api/CuratedContent.java

```diff
@@ -1,25 +1,26 @@
 package com.wikia.webdriver.common.core.api;
 
-import com.wikia.webdriver.common.core.annotations.User;
-import com.wikia.webdriver.common.core.configuration.Configuration;
-import com.wikia.webdriver.common.core.url.UrlBuilder;
+import java.util.ArrayList;
 
 import org.apache.http.message.BasicNameValuePair;
 
-import java.util.ArrayList;
+import com.wikia.webdriver.common.core.annotations.User;
+import com.wikia.webdriver.common.core.configuration.Configuration;
+import com.wikia.webdriver.common.core.url.UrlBuilder;
 
 public class CuratedContent extends ApiCall {
 
-  public CuratedContent() {
-  }
+  public CuratedContent() {}
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Move this closing curly brace to the next line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3ARightCurlyBraceStartLineCheck)